internal/mode/static/nginx/config/servers_test.go

```diff
@@ -1711,17 +1704,17 @@ func TestCreatePathForMatch(t *testing.T) {
 		panic    bool
 	}{
 		{
-			expected: "/path_prefix_route1",
```

Comment:
This test can be simplified now since the function no longer accepts `pathType`. You can just check that `g.Expect(createPathForMatch(0,1)).To(Equal("@rule0-route1"))`

Reply:
updating

Reply:
will probably remove the test `TestCreatePathForMatch` , since there are other checks for path in `server_test.go` and remove the function too `createPathForMatch`